Makefile

```diff
@@ -20,9 +20,9 @@ generate:
 rebuild: generate build test
 
 check: test
-test:
-	go test -race ./...
-	go vet ./...
 	# tests should fail if go generate or go fmt results in uncommitted code
 	git status --porcelain
 	/bin/bash -c 'test $$(git status --porcelain | wc -l) == 0'
+test:
+	go test -race ./...
+	go vet ./...
```

Comment:
Moving things around here... is mainly because I like to run `make test` locally while I have changes in local folder... so the output from `git status --porcelain` is annoying (and makes test fail, because I haven't committed yet).